aya-bpf-macros/src/lib.rs

```diff
@@ -118,6 +118,44 @@ pub fn cgroup_skb(attrs: TokenStream, item: TokenStream) -> TokenStream {
         .into()
 }
 
+/// Marks a function as a [`CgroupSockAddr`] eBPF program. 
+///
+/// [`CgroupSockAddr`] programs can be used to inspect or modify socket addresses passed to
+/// various syscalls within a [cgroup]. They can be attached to a number of different
+/// places as described in [`CgroupSockAddrAttachType`].
+///
+/// [cgroup]: https://man7.org/linux/man-pages/man7/cgroups.7.html
+/// [`CgroupSockAddrAttachType`]: ../aya/programs/cgroup_sock_addr/enum.CgroupSockAddrAttachType.html
+/// [`CgroupSockAddr`]: ../aya/programs/cgroup_sock_addr/struct.CgroupSockAddr.html
+/// 
+/// # Arguments
+/// 
+/// * `attach_type` - The type of syscall to attach to. See [`CgroupSockAddrAttachType`].
```

Comment:
For arguments we usually stick to this convention:

https://www.tangramvision.com/blog/making-great-docs-with-rustdoc#-arguments

So we don't list `Arguments` explicitly unless really necessary. And we usually prefer mentioning arguments in the general description, which, I believe, can be done here.